lib/mongoid/errors/invalid_estimated_count_criteria.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+# encoding: utf-8
+
+module Mongoid
+  module Errors
+
+    # Creates the exception raised when trying to call estimated_count
+    # on a filtered criteria.
+    #
+    # @param [ String ] class_name The class name.
```

Comment:
What class is this the name of?

Reply:
The `klass` of the criteria used to call estimated count... I can put that in the docstring